client/src/Pages/LoginForm.jsx

```diff
@@ -27,7 +34,12 @@ const LoginForm = () => {
                     .max(15, "Very funny, please put just your first name" )
                     .required("You forgot your name... you fool!")
             })}
-            onSubmit={(values, {setSubmitting, resetForm}) => {
+            onSubmit={(values) => {
```

Comment:
If you wanted to send the name up you could:
1. Create a `name` reference in `LoginForm` with `useRef` (I just tested with `useState` and for some reason it didn't work).
2. In this `onSubmit` function, set `name.current` to the name.
3. In `getRole`, include `name.current` in the data you send to `onRole` (line 11).

More info on `useRef` [here](https://reactjs.org/docs/hooks-reference.html#useref)

Reply:
Perfect, ill make an issue with this and make it a stretch goal